.github/workflows/continuous-integration.yml

```
@@ -17,53 +17,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-13, macos-latest, windows-2022]
-        gfortran-version: [12, 13, 14]
-        gfortran-from: [system, conda]
+        rte-kernels: [default, accel]
         fpmodel: [DP, SP]
+        build-type: [Debug, RelWithDebugInfo]
+        # Add Release config?
+        # Exclude RelWithDebugInfo/accel combinations?
         exclude:
-          - os: ubuntu-24.04
-            gfortran-from: conda
-          - os: macos-13
-            gfortran-from: system
-          - os: macos-13
-            gfortran-version: 14
-            gfortran-from: conda
-          - os: macos-latest
-            gfortran-from: system
-          - os: macos-latest
-            gfortran-version: 14
-            gfortran-from: conda
-          - os: windows-2022
+        - build-type: RelWithDebugInfo
+          rte-kernels: accel
+        #
+        # Add debug flags
+        #
         include:
-          - os: ubuntu-24.04
-            gfortran-version: 13
-            gfortran-from: conda
-            fpmodel: DP
-          - os: windows-2022
-            gfortran-version: 13
-            gfortran-from: conda
-            fpmodel: DP
-          - os: windows-2022
-            gfortran-version: 13
-            gfortran-from: conda
-            fpmodel: SP
-          - os: windows-2022
-            gfortran-version: 14
-            gfortran-from: conda
-            fpmodel: DP
-          - os: windows-2022
-            gfortran-version: 14
-            gfortran-from: conda
-            fpmodel: SP
+        - build_type: Debug
+          fflags: -std=f2008 -fbounds-check -fmodule-private -fimplicit-none -finit-real=nan
+
     env:
-      FC: gfortran-${{ matrix.gfortran-version }}
-      FFLAGS: "-m64 -std=f2008 -march=native -fbounds-check -fmodule-private -fimplicit-none -finit-real=nan"
       CMAKE_BUILD_PARALLEL_LEVEL: 8
       VERBOSE:
       CTEST_PARALLEL_LEVEL: 8
       CTEST_OUTPUT_ON_FAILURE: 1
-    runs-on: ${{ matrix.os }}
+      FC: gfortran-14
+    runs-on: ubuntu-24.04 # latest?
```

No. We don't want the testing environment to change "randomly" at some point.
```suggestion
    runs-on: ubuntu-24.04
```